Makefile

```diff
@@ -210,7 +210,9 @@ integration-tests-v2: $(ENVTEST) ## Run tests with reconciler V2
 
 .PHONY: e2e-tests
 e2e-tests: manifests $(KUSTOMIZE) ## Run E2E tests and destroy environment stacks after tests complete. To run locally, complete pre-reqs (see docs/how-to-contribute.md) and prepend command with `aws-vault exec sso-agent-sandbox-account-admin --`. E.g. `aws-vault exec sso-agent-sandbox-account-admin -- make e2e-tests`.
-	KUBEBUILDER_ASSETS="$(ROOT)/bin/$(PLATFORM)/" go test -C test/e2e --tags=e2e github.com/DataDog/datadog-operator/e2e -v -timeout 1h -coverprofile cover_e2e.out
+	cd test/e2e && go get github.com/DataDog/datadog-agent/test/new-e2e@1cc7604f4b6b35a230b3dc1226d35b86034a832a
+	cd $(ROOT)
```

Comment:
Note: these lines are for testing in the CI. They'll be removed when `test/e2e/go.mod` is updated with the new version when it is available.